.github/workflows/build-nethermind-packages.yml

```diff
@@ -9,50 +9,45 @@ on:
 
 jobs:
   build-nethermind:
-    name: Building Nethermind Packages
+    name: Build Nethermind packages
     runs-on: ubuntu-latest
     env: 
       RELEASE_DIRECTORY: /home/runner/work/nethermind/nethermind
       LIN_RELEASE: nethermind-lin-x64
+      LIN_ARM64_RELEASE: nethermind-lin-arm64
+      WIN_RELEASE: nethermind-win-x64
       OSX_RELEASE: nethermind-osx-x64
       OSX_ARM64_RELEASE: nethermind-osx-arm64
-      WIN_RELEASE: nethermind-win-x64
-      LIN_ARM64_RELEASE: nethermind-lin-arm64
-      LINUX: linux-x64
-      OSX: osx-x64
-      OSX_ARM64: osx-arm64
-      WIN10: win10-x64
-      LINUX_ARM64: linux-arm64
     steps:
-    - name: Checking out Nethermind repository
+    - name: Check out Nethermind repository
       uses: actions/checkout@v3
       with:
         submodules: recursive
         path: nethermind
-    - name: Checking out Nethermind Launcher repository
+    - name: Check out Nethermind Launcher repository
       uses: actions/checkout@v3
       with:
         repository: NethermindEth/nethermind.launcher
         path: launcher
-    - name: Setting up Node.js
+    - name: Set up Node.js
       uses: actions/setup-node@v3
       with:
-        node-version: "14"
-    - name: Setting up Build Environment
+        node-version: '14'
+    - name: Set up build environment
       run: |
         npm i pkg @vercel/ncc -g
-    - name: Setting up dotnet
-      uses: actions/setup-dotnet@v2
+    - name: Set up .NET
+      uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: '6.0.x'
-    - name: Setting up QEMU
+        global-json-file: ./nethermind/global.json
```

Comment:
Can you make this the same for each setup-dotnet action? This way, we can be sure which version is being used.

Reply:
That would defeat the whole point of not having a setting under each action. We specify it when we specify the repo path. Otherwise, I'd go with `dotnet-version`. Still considering.